app/lib/sprites/SpriteBuilder.coffee

```diff
@@ -143,7 +143,19 @@ module.exports = class SpriteBuilder
 
     for group, config of colorConfig
       continue unless colorGroups[group] # color group not found...
-      @buildColorMapForGroup(colorGroups[group], config)
+      if @thangType.get('ozaria')
+        @buildOzariaColorMapForGroup(colorGroups[group], config)
+      else
+        @buildColorMapForGroup(colorGroups[group], config)
+
+  # Simpler Ozaria color mapper.
+  # Instead of color shifting we apply the color directly.
+  buildOzariaColorMapForGroup: (shapes, config) ->
+    return unless shapes.length
+    for shapeKey in shapes
+      shape = @shapeStore[shapeKey]
+      continue if not shape.fc?
+      @colorMap[shapeKey] = hslToHex([config.hue, config.saturation, config.lightness])
```

Comment:
I'm not quite sure what color map is / what this is doing.  Code looks generally sound but can't comment on its functionality.  Maybe a shortish comment here describing the intention would provide some context?  Or a reference to other color map code that could provide it?

Reply:
Fair point. The colorMap key is a shape, and the value is the new color to use.

This method creates a large object which can be queried when the shapes are being drawn to see if an alternative color should be used.

Will add comment.